meta/statement_executor.go

```diff
@@ -26,7 +26,7 @@ type StatementExecutor struct {
 		CreateUser(name, password string, admin bool) (*UserInfo, error)
 		UpdateUser(name, password string) error
 		DropUser(name string) error
-		SetPrivilege(username, database string, p influxql.Privilege) error
+		SetPrivilege(username, database string, p influxql.Privilege, admin bool) error
```

Comment:
Just wondering -- we couldn't extend the meaning of `influxql.Privilege` to include an admin level, could we?